### PR TITLE
Test k8s liveness readiness integration

### PR DESCRIPTION
**Pull request by DmitriyMolch**

### CODE REVIEW

File: api/src/services/healthService.js [hunk 1]
```diff
@@ -106,27 +106,29 @@ async function liveness(req, res) {
...
-  }
-
-  if (!ethNetworkService.initialized()) {
-    errors.push('Ethereum network service is not initialized yet');
-  }
+    try {
+      const client = await getClient();
+      client.end();
+    } catch (error) {
+      console.error(error);
```
Comment (MRamanenkau) [inline api/src/services/healthService.js:113]: Why do we need console.log?

**Review by MRamanenkau — APPROVED** [review]

File: api/src/services/healthService.js (hunk 1 above)
Comment (DmitriyMolch) [reply api/src/services/healthService.js:113]: We need to see in the logs which exact database error happened.

Comment (DmitriyMolch) [reply api/src/services/healthService.js:113]: When we deploy a pod the `readiness` API is not publicly available if database error happens this is why we need to see it in the logs.

File: api/src/services/healthService.js [hunk 2]
```diff
@@ -106,27 +106,29 @@ async function liveness(req, res) {
...
-  if (!cereNetworkService.initialized()) {
-    errors.push('Cere network service is not initialized yet');
-  }
-
-  if (!ethNetworkService.initialized()) {
-    errors.push('Ethereum network service is not initialized yet');
-  }
+    try {
+      const client = await getClient();
+      client.end();
```
Comment (AndreiNavoichyk) [inline api/src/services/healthService.js:111]: You should move this logic in the `finally` block to guarantee it will execute.

Comment (DmitriyMolch) [reply api/src/services/healthService.js:111]: fixed

**Pull request merged**